packages/core/core-flows/src/order/workflows/create-return.ts

```diff
@@ -7,28 +8,27 @@ import {
   WithCalculatedPrice,
 } from "@medusajs/types"
 import {
-  createStep,
-  createWorkflow,
-  transform,
-  WorkflowData,
-} from "@medusajs/workflows-sdk"
-import { createLinkStep, useRemoteQueryStep } from "../../common"
-import {
-  arrayDifference,
   ContainerRegistrationKeys,
-  isDefined,
   MathBN,
   MedusaError,
-  Modules,
+  OrderStatus,
+  arrayDifference,
+  isDefined,
   remoteQueryObjectFromString,
 } from "@medusajs/utils"
+import {
+  WorkflowData,
+  createStep,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { createLinkStep, useRemoteQueryStep } from "../../common"
+import { createFulfillmentWorkflow } from "../../fulfillment"
 import { updateOrderTaxLinesStep } from "../steps"
 import { createReturnStep } from "../steps/create-return"
-import { createFulfillmentWorkflow } from "../../fulfillment"
 
 function throwIfOrderIsCancelled({ order }: { order: OrderDTO }) {
-  // TODO: need work, check canceled
-  if (false /*order.canceled_at*/) {
+  if (order.status === OrderStatus.CANCELED) {
```

Comment:
note: as mentioned on the other pr we might move those functions to the utils dir next to the order workflow, so that we can create our different validation step in those workflow using those utils functions. wdyt?